src/util/chains.ts

```diff
@@ -46,8 +54,30 @@ export enum ChainName {
   OPTIMISTIC_KOVAN = 'optimism-kovan',
```

Comment:
these match infura according to comments, but then we have multiple rpc urls anyway, strange

Reply:
its cause we use these in the routing api but yeah, we need to probably just use the project id in SOR

Reply:
i think i prefer using separate env variables, so we could use an endpoint provided by a layer 2 if we wanted, just thought it's a bit strange the comment that it matches without the reason